mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -763,11 +764,13 @@ private[python] class PythonMLLibAPI extends Serializable {
       maxBins = maxBins,
       categoricalFeaturesInfo = categoricalFeaturesInfo.asScala.toMap)
     val cached = data.rdd.persist(StorageLevel.MEMORY_AND_DISK)
+    // Only done because methods below want an int, not an optional Long
+    val intSeed = getSeedOrDefault(seed).toInt
```

Comment:
I was hoping to be consistent with the other places in the class that needed to maybe get a random seed. You're saying that the lower 32 bits won't necessarily be as random? I would think that theoretically they are. In practice we don't need cryptographic-strength guarantees here anyway.